Adafruit_L3GD20_U.cpp

```diff
@@ -64,6 +64,7 @@ byte Adafruit_L3GD20_Unified::read8(byte reg)
   #endif
   Wire.endTransmission();
   Wire.requestFrom((byte)L3GD20_ADDRESS, (byte)1);
+  while (!Wire.available()); // Wait for data to arrive.
```

Comment:
Yup... I didn't entirely know what I was doing at the time, and I thought that fixed my problem I was having at the time. It didn't. I never did figure out the problem, either. Sorry.